bot/discordbot.py

```diff
@@ -174,6 +178,94 @@ async def hello(interaction: discord.Interaction):
     await interaction.response.send_message(f"Hey {interaction.user.mention}! This is a slash command!")
 
 
+# Function for server admin to register credential info into Developer Console using '/register' command in the "get-verified" channel
+@bot.tree.command(name="register")
+async def register(interaction: discord.Interaction, email:str):
+
+    #Check if the command is used in the "get-verified" channel
+    channels = discord.utils.get(interaction.guild.channels, name='get-verified')
+    if not channels:
+        channels = await interaction.guild.create_text_channel('get-verified')
```

Comment:
It will be nice to wrap this in a `try-catch` statement. Should in case the `get-verified` channel can not be created, a message will be returned to the user.